### DISCUSSION TITLE
Discord vs Slack?

### DISCUSSION THREAD

**Discussion by phifa** [keystonejs/keystone#8166] [Polls]
80% of my web related Open Source projects are on Discord, not on Slack. Why is that? Would there be any advantages for keystone to move to Discord?

**Comment by flexdinesh** [comment]
We have thought about it @phifa. Keystone and the Keystone community has been on Slack long before Discord became popular. As much as we like Discord, moving an entire community from Slack to discord is not an easy feat and we don't want to introduce this disrupting change to the community. Slack works well for us and as long as it works well for us, we intend to stay on Slack.

**Reply by phifa** [reply]
Cool makes sense!

**Reply by phifa** [reply]
I saw today, Prisma now also moved to Discord. Seems to be the new place for OS. is KeystoneJS still under active development and would it be worth to do the move or do you still think to big of an undertaking @flexdinesh ? 

<img width="663" alt="Screenshot 2023-06-29 at 09 56 33" src="https://github.com/keystonejs/keystone/assets/3061129/52327885-80fd-4bb5-8223-d735373d2616">
